src/libraries/Common/src/Interop/Windows/SspiCli/Interop.SSPI.cs

```diff
@@ -315,6 +316,23 @@ public SecBufferDesc(int count)
             }
         }
 
+        [StructLayout(LayoutKind.Sequential)]
+        internal unsafe struct SecPkgCred_ClientCertPolicy
+        {
+            public int dwFlags;
```

Comment:
```suggestion
            public uint dwFlags;
```
Nit: To match unmanaged def